enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/analytics/AnalyticsFilter.java

```diff
@@ -168,8 +170,12 @@ public void handleSuccessRequest(RequestContext requestContext) {
                     ConfigHolder.getInstance().getEnvVarConfig().getEnforcerRegionId());
 
             // As in the matched API, only the resources under the matched resource template are selected.
+            ArrayList<String> resourceTemplate = new ArrayList<>();
+            for (ResourceConfig resourceConfig : requestContext.getMatchedResourcePaths()) {
+                resourceTemplate.add(resourceConfig.getPath());
+            }
             requestContext.addMetadataToMap(MetadataConstants.API_RESOURCE_TEMPLATE_KEY,
-                    requestContext.getMatchedResourcePath().getPath());
+                    String.join(",", resourceTemplate));
```

Comment:
Do we add multiple resource Paths here?

Reply:
yes. for graphql, it is

Reply:
As per Amali, the choreo analytics splits the resources and upgrade accordingly.